specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/stable/2022-12-12/iotdps.json

```diff
@@ -0,0 +1,2571 @@
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "state",
+            "modelAsString": true
+          }
+        },
+        "publicNetworkAccess": {
+          "type": "string",
+          "description": "Whether requests from Public Network are allowed",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "PublicNetworkAccess"
+          }
+        },
+        "ipFilterRules": {
+          "description": "The IP filter rules.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IpFilterRule"
+          }
+        },
+        "privateEndpointConnections": {
+          "description": "Private endpoint connections created on this IotHub",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnection"
+          }
+        },
+        "provisioningState": {
+          "description": "The ARM provisioning state of the provisioning service.",
+          "type": "string"
+        },
+        "iotHubs": {
+          "type": "array",
+          "description": "List of IoT hubs associated with this provisioning service.",
+          "items": {
+            "$ref": "#/definitions/IotHubDefinitionDescription"
+          }
+        },
+        "allocationPolicy": {
+          "enum": [
+            "Hashed",
+            "GeoLatency",
+            "Static"
+          ],
+          "type": "string",
+          "description": "Allocation policy to be used by this provisioning service.",
+          "x-ms-enum": {
+            "name": "AllocationPolicy",
+            "modelAsString": true
+          }
+        },
+        "serviceOperationsHostName": {
+          "description": "Service endpoint for provisioning service.",
+          "type": "string",
+          "readOnly": true
+        },
+        "deviceProvisioningHostName": {
+          "type": "string",
+          "description": "Device endpoint for this provisioning service.",
+          "readOnly": true
+        },
+        "idScope": {
+          "type": "string",
+          "description": "Unique identifier of this provisioning service.",
+          "readOnly": true
+        },
+        "authorizationPolicies": {
+          "description": "List of authorization keys for a provisioning service.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SharedAccessSignatureAuthorizationRule[AccessRightsDescription]"
+          }
+        },
+        "enableDataResidency": {
+          "type": "boolean",
+          "description": "Optional.\nIndicates if the DPS instance has Data Residency enabled, removing the cross geo-pair disaster recovery."
+        },
+        "portalOperationsHostName": {
+          "type": "string",
+          "description": "Portal endpoint to enable CORS for this provisioning service."
```

Comment:
>Portal endpoint to enable CORS for this provisioning service. [](http://example.com/codeflow?start=26&length=61)

Just curious, if you are looking for Azure Portal endpoint here, using which the client made this call? FYI: standard client headers which are passed to RPs by ARM, including "referrer" header: [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#proxy-request-header-modifications](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#proxy-request-header-modifications).

Please remove ARMChangesRequest label, after providing the answer to this question, unless you have some further changes to make.

Reply:
Not sure I fully understand the question, but by Portal, I meant an internal team within Azure IoT that will consume the endpoint that we are adding.

Reply:
Isn't the referer header passed to RPs by ARM is... an ARM address?